package.json

```diff
@@ -30,7 +30,7 @@
   "homepage": "https://github.com/mdn/browser-compat-data#readme",
   "devDependencies": {
     "ajv": "^6.10.0",
-    "better-ajv-errors": "^0.6.1",
+     "better-ajv-errors": "^0.6.4",
```

Comment:
For whatever reason, GitHub decided to insert a space here, despite the suggestion displaying it without said space.